mephisto/scripts/local_db/gh_actions/auto_generate_blueprint.py

```diff
@@ -0,0 +1,78 @@
+from distutils import extension
+from mephisto.client.cli_commands import get_wut_arguments
+from rich import print
+from mdutils.mdutils import MdUtils
+from copy import deepcopy
+
+
+def create_blueprint_info(blueprint_file, arg_dict):
+    blueprint_file.new_paragraph(arg_dict["desc"].strip())
+    arg_keys = list(arg_dict["args"].keys())
+    first_arg = list(arg_dict["args"].keys())[0]
+    first_arg_keys = list(arg_dict["args"][first_arg].keys())
+
+    list_of_strings = deepcopy(first_arg_keys)
+
+    for cur_key in arg_keys:
+        extension_list = []
+        for first_arg_key in first_arg_keys:
+            item_content = arg_dict["args"][cur_key][first_arg_key]
+            item_to_append = (
+                "".join(item_content.splitlines())
+                if isinstance(item_content, str)
+                else item_content
+            )
+            print(item_to_append)
+            extension_list.append(item_to_append)
+        list_of_strings.extend(extension_list)
+
+    blueprint_file.new_line()
+
+    # Add 1 to rows to account for header row
+    blueprint_file.new_table(
+        rows=len(arg_keys) + 1,
+        columns=len(first_arg_keys),
+        text=list_of_strings,
+        text_align="left",
+    )
+
+
+def main():
+    blueprint_file = MdUtils(
+        file_name="../../../../docs/web/docs/reference/Blueprints.md",
```

Comment:
Any way to make this path cleaner?

Reply:
I'm not sure if there's an importable python constant/method floating around the codebase that could get you the root dir. @JackUrb might know